src/services/contentProgress.js

```diff
@@ -13,7 +13,7 @@ const STATE_COMPLETED = 'completed';
 const DATA_KEY_STATUS = 's';
 const DATA_KEY_PROGRESS = 'p';
 const DATA_KEY_RESUME_TIME = 't';
-export let dataContext = new DataContext(ContentProgressVersionKey, fetchContentProgress());
+export let dataContext = new DataContext(ContentProgressVersionKey, fetchContentProgress);
```

Comment:
changed this to resolve "[TypeError: this.fetchDataFunction is not a function (it is Object)]"